repos/system_upgrade/common/models/systemdservices.py

```diff
@@ -0,0 +1,9 @@
+from leapp.models import fields, Model
+from leapp.topics import SystemInfoTopic
+
+
+class SetSystemdServicesStateTask(Model):
```

Comment:
Use rather `Tasks` instead of `Task` as it's more common with other models. Also let's be rather more generic about the name. What if we would like to add in future (no idea what it could be now) more tasks then just enable/disable? Let's keep doors opened for future extensions.
```suggestion
class SystemdServicesTasks(Model):
```

Also I am thinking about `TargetSystemdServicesTasks` to make it clear we speak just about tasks handling services on the target system. Just in case we would like to do something around services even inside the upgrade initramfs. but for now, let's stick to the shorter one.